contributors/devel/api-conventions.md

```diff
@@ -1240,6 +1240,8 @@ latter if the label/annotation is specific to `foo`
 the resource doesn't need to know about, experimental fields that aren't
 intended to be generally used API fields, etc. Beware that annotations aren't
 automatically handled by the API conversion machinery.
+  - The key name of [taint](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) should also follow the naming conversion of annotation.
+    - For instance, prefer `foo.alpha.kubernetes.io/foo-bar` over `foo.alpha.kubernetes.io/fooBar`
```

Comment:
also remove `alpha` in annotation; and not only the name of taint, but also for other annotation name, I think.

Reply:
ack